config/sync/geocoder.geocoder_provider.googlemaps.yml

```diff
@@ -6,7 +6,7 @@ id: googlemaps
 label: GoogleMaps
 plugin: googlemaps
 configuration:
-  apiKey: '<placeholder>'
+  apiKey: ''
```

Comment:
Harmless change, but you can probably omit this from the PR as it looks auto-exported.

Reply:
👍🏻 - oopsy